packages/base/src/util/InvisibleMessage.js

```diff
@@ -0,0 +1,48 @@
+import InvisibleMessageMode from "../types/InvisibleMessageMode.js";
+import getSingletonElementInstance from "./getSingletonElementInstance.js";
+
+const styles = `position: absolute;
+	clip: rect(1px,1px,1px,1px);
+	user-select: none;
+	left: -1000px;
+	top: -1000px;
+	pointer-events: none;`;
+
+const politeSpan = document.createElement("span");
```

Comment:
This code must be synchronized with the lifecycle of the framework. You can achieve this in one of two ways:
 - Make the creation of these spans lazy (during the call of the `announce` function - an only create the span that is announced - polite or assertive)
 - Export another function from this file, and call it in `boot.js` to create the spans safely after the HTML is loaded..
 
 I prefer the first.